src/controller/replication/flow/mock.go

```diff
@@ -15,7 +15,6 @@
 package flow
 
 import "github.com/goharbor/harbor/src/pkg/reg/adapter"
-
```

Comment:
same comment as this one:
* https://github.com/goharbor/harbor/pull/17131/files#r923010380